index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Peter Okorafor - Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="topnav">
+            <a href="#home" class="logo">Peter Okorafor</a>
+        </div>
+    </header>
+
+    <div class="container">        
+        <p class="container headline">Hey There.<br>I'm Peter</p>
```

Comment:
Kindly change your title tag to `<h1>`, the main title of a page should always use `<h1>` to follow good practices :100:

Reply:
Should have seen that too 😄